ioctl/newcmd/did/didgethash.go

```diff
@@ -42,20 +42,15 @@ func NewDidGetHash(client ioctl.Client) *cobra.Command {
 		Args:  cobra.ExactArgs(2),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			cmd.SilenceUsage = true
-			contract, err := client.Address(args[0])
+			addr, err := alias.IOAddress(client, args[0])
 			if err != nil {
 				return errors.Wrap(err, "failed to get contract address")
 			}
-			addr, err := address.FromString(contract)
-			if err != nil {
-				return errors.Wrap(err, "invalid contract address")
-			}
 			bytecode, err := _didABI.Pack(_getHashName, []byte(args[1]))
 			if err != nil {
 				return errors.Wrap(err, "invalid bytecode")
 			}
-
-			result, err := action.Read(client, addr, "0", bytecode, contract, 20000000)
+			result, err := action.Read(client, addr, "0", bytecode, action.SignerFlagDefault, action.GasLimitFlagDefault)
```

Comment:
why is `contract` changed to `action.SignerFlagDefault`

Reply:
`contract` is wrong parameter.  `contract` is corresponding to `addr`. `action.SignerFlagDefault` is the right parameter according to `cmd/` logic